sdk/storage/azblob/internal/testcommon/cpk.go

```diff
@@ -0,0 +1,37 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+// Contains common helpers for TESTS ONLY
+package testcommon
+
+import "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob/blob"
+
+var testEncryptedKey = "MDEyMzQ1NjcwMTIzNDU2NzAxMjM0NTY3MDEyMzQ1Njc="
+var testEncryptedHash = "3QFFFpRA5+XANHqwwbT4yXDmrT/2JaLt/FKHjzhOdoE="
+var testEncryptionAlgorithm = blob.EncryptionAlgorithmTypeAES256
+var TestCPKByValue = blob.CpkInfo{
+	EncryptionKey:       &testEncryptedKey,
+	EncryptionKeySHA256: &testEncryptedHash,
+	EncryptionAlgorithm: &testEncryptionAlgorithm,
+}
+
+var testInvalidEncryptedKey = "mumbojumbo"
+var testInvalidEncryptedHash = "mumbojumbohash"
+var TestInvalidCPKByValue = blob.CpkInfo{
+	EncryptionKey:       &testInvalidEncryptedKey,
+	EncryptionKeySHA256: &testInvalidEncryptedHash,
+	EncryptionAlgorithm: &testEncryptionAlgorithm,
+}
+
+var testEncryptedScope = "blobgokeytestscope"
+var TestCPKByScope = blob.CpkScopeInfo{
+	EncryptionScope: &testEncryptedScope,
+}
+
+var testInvalidEncryptedScope = "mumbojumbo"
```

Comment:
Make this a different value from the testInvalidEncryptedKey 
```suggestion
var testInvalidEncryptedScope = "mumbojumboscope"
```

Reply:
I can clean this up, however the overall goal in this PR is to not make any functional test changes (so they don't get lost in the refactoring).